src/commands/createNewProject/validateFuncCliInstalled.ts

```diff
@@ -0,0 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as opn from 'opn';
+import { MessageItem, OutputChannel } from 'vscode';
+import { callWithTelemetryAndErrorHandling, DialogResponses, IActionContext, IAzureUserInput } from 'vscode-azureextensionui';
+import { Platform } from '../../constants';
+import { ext } from '../../extensionVariables';
+import { localize } from '../../localize';
+import { getFuncExtensionSetting, updateGlobalSetting } from '../../ProjectSettings';
+import { cpUtils } from '../../utils/cpUtils';
+
+export async function validateFuncCliInstalled(ui: IAzureUserInput, outputChannel: OutputChannel): Promise<void> {
+    await callWithTelemetryAndErrorHandling('azureFunctions.validateFuncCliInstalled', ext.reporter, undefined, async function (this: IActionContext): Promise<void> {
+        this.suppressErrorDisplay = true;
+        const settingKey: string = 'showFuncInstallation';
+        if (getFuncExtensionSetting<boolean>(settingKey)) {
+            if (!(await funcCliInstalled()) && (await brewOrNpmInstalled()) && process.platform !== Platform.Linux) {
+                // https://github.com/Microsoft/vscode-azurefunctions/issues/311
+                const input: MessageItem = await ui.showWarningMessage(localize('installFuncCli', 'You need Azure Functions Core Tools CLI to locally debug your functions.  Run {0} now?', getCommandForPlatform()), DialogResponses.yes, DialogResponses.dontWarnAgain, DialogResponses.skipForNow);
```

Comment:
Remove "CLI". It's just called "Azure Functions Core Tools" in all the documentation